### ISSUE TITLE
Discrepancy Between Results in the Repository and Roboflow100

### ISSUE THREAD

**Issue by JiahuiChen-GitHub** [Deci-AI/super-gradients#1337]
### 💡 Your Question

Hi, I've been using your Yolo-nas and appreciate your effort. However, I've noticed differences between the results presented in the repository and those reported in the Roboflow ([paper](https://arxiv.org/abs/2211.13523) or [repo](https://github.com/roboflow/roboflow-100-benchmark)). Could you kindly clarify the reasons behind these discrepancies?

The [paper](https://arxiv.org/abs/2211.13523) reports the average mAP of yolov5 and yolov7. The [repo](https://github.com/roboflow/roboflow-100-benchmark) presents the average mAP of yolov5, yolov7 and yolov8 on the 88 datasets. Notably, the results presented in your repository are better than those reported in both the paper and the Roboflow benchmark.

I would like to know how I can reproduce the results you gave on the RF100 dataset.

Best regards.

**Comment by shaydeci** [comment]
Hello @JiahuiChen-GitHub, the results reported for the other Yolo variants were taken from [here](https://github.com/roboflow/roboflow-100-benchmark/blob/8587f81ef282d529fe5707c0eede74fe91d472d0/metadata/datasets_stats.csv).
For a fair comparison, we evaluated YoloNAS only on the datasets that had a reported score.

For reproduction, follow the [RF100 training steps for each dataset](https://docs.deci.ai/super-gradients/documentation/source/YoloNASQuickstart.html#train-on-rf100).

**Comment by JiahuiChen-GitHub** [comment]
Thanks for the answer!

**Comment by JiahuiChen-GitHub** [comment]
Hello, we have another new question. In this blog, you showcased the performance of yolonas-s and yolonas-m on the rf100 dataset. Could you please share the results of both models across various subclasses of rf100?

**Comment by BloodAxe** [comment]
I don't think we have them computed during training. And because of that they per class scores are not logged to experiment monitoring platform. 
I'm not sure we kept best checkpoints for all 100 datasets either. 
So my best recommendation for you to take out fine tune recipe for roboflow and train model that you are interested in and train it with "include_classwise_ap" option enabled in DetectionMetris class

**Comment by JiahuiChen-GitHub** [comment]
<img width="946" alt="截屏2023-11-11 16 56 57" src="https://github.com/Deci-AI/super-gradients/assets/77907090/d8470b75-c035-42d8-8243-24b0c08dc0bd">

Sorry, I meant the mAP per category in this image.

**Comment by BloodAxe** [comment]
Like absolute values? Since this is image represents exactly what you asking

**Comment by JiahuiChen-GitHub** [comment]
Yes, we are doing a study on real-time object detectors and would like to get some absolute values for a fair comparison.

**Comment by JiahuiChen-GitHub** [comment]
Could you please provide the absolute values of yolonas-s and yolonas-m models in this image?
![](https://user-images.githubusercontent.com/77907090/282231936-d8470b75-c035-42d8-8243-24b0c08dc0bd.png)
